src/app/modules/item/pages/item-progress/item-progress.component.html

```diff
@@ -0,0 +1,2 @@
+<h1>Progress</h1>
+<router-outlet></router-outlet>
```

Comment:
Try to have a correct style directly. Normally, that should not be too difficult as it has been done for group pages. (and the design branch)